Added because of: https://bugzilla.suse.com/show_bug.cgi?id=1099800
This patch was proposed by Simon Quigly and was declined by upstream.
See discussion at: https://github.com/lxqt/lxqt-session/pull/126/files

From e76f0ebf14b2d54495b3fa27fe751b229b15f6d8 Mon Sep 17 00:00:00 2001
From: Simon Quigley <tsimonq2@ubuntu.com>
Date: Fri, 25 May 2018 22:57:21 -0500
Subject: [PATCH] Ensure all paths are added to $XDG_CONFIG_DIRS.

---
 startlxqt.in | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/startlxqt.in b/startlxqt.in
index ae0569e..58b2b25 100755
--- a/startlxqt.in
+++ b/startlxqt.in
@@ -26,9 +26,11 @@ export XDG_DATA_DIRS
 if [ -z "$XDG_CONFIG_DIRS" ]; then
     export XDG_CONFIG_DIRS="@PREDEF_XDG_CONFIG_DIRS@"
 else
-    if ! contains "$XDG_CONFIG_DIRS" '@LXQT_ETC_XDG_DIR@'; then
-        XDG_CONFIG_DIRS="$XDG_CONFIG_DIRS:@LXQT_ETC_XDG_DIR@"
-    fi
+    for directory in "/etc" "@LXQT_ETC_XDG_DIR@" "/usr/share"; do
+        if ! contains "$XDG_CONFIG_DIRS" "$directory"; then
+            XDG_CONFIG_DIRS="$XDG_CONFIG_DIRS:$directory"
+        fi
+    done
 fi
 
 if [ -z "$XDG_CACHE_HOME" ]; then
